modules/components/org.wso2.carbon.event.simulator.core/pom.xml

```diff
@@ -0,0 +1,244 @@
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.json.wso2</groupId>
+            <artifactId>json</artifactId>
+        </dependency>
+        <!--<dependency>-->
+            <!--<groupId>org.apache.axis2.wso2</groupId>-->
+            <!--<artifactId>axis2-client</artifactId>-->
+            <!--<version>${orbit.axis2.client.version}</version>-->
+        <!--</dependency>-->
+        <!--<dependency>-->
+            <!--<groupId>org.wso2.apache.httpcomponents</groupId>-->
+            <!--<artifactId>httpmime</artifactId>-->
+            <!--<version>${apache.httpcomponents.version}</version>-->
+        <!--</dependency>-->
+        <dependency>
+            <groupId>org.wso2.orbit.org.antlr</groupId>
+            <artifactId>antlr4-runtime</artifactId>
+        </dependency>
+
+       <!-- Dependencies for CSV event simulations begins here-->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-csv</artifactId>
+            <version>${apache.commons.csv.version}</version>
+        </dependency>
+        <!-- Dependencies for CSV event simulations ends here-->
+
+        <!--Dependencies for random event simulation begins here-->
+        <dependency>
+            <groupId>com.github.javafaker</groupId>
+            <artifactId>javafaker</artifactId>
+            <version>${javafaker.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.azakordonets</groupId>
+            <artifactId>fabricator_2.10</artifactId>
+            <version>${fabricator.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.mifmif</groupId>
+            <artifactId>generex</artifactId>
+            <version>${generex.version}</version>
+        </dependency>
+        <!--Dependencies for random event simulation ends here-->
+
+        <!--Dependencies for database feed simulation begins here-->
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>${mysql.connector.version}</version>
+        </dependency>
+
+
+        <!--<dependency>-->
+            <!--<groupId>org.scala-lang</groupId>-->
+            <!--<artifactId>scala-compiler</artifactId>-->
+            <!--<version>2.11.0</version>-->
+        <!--</dependency>-->
+
+    </dependencies>
+
+    <properties>
+        <maven.findbugsplugin.version.exclude>findbugs-exclude.xml</maven.findbugsplugin.version.exclude>
+        <microservice.mainClass>Application</microservice.mainClass>
+        <export.package>
+            !org.wso2.eventsimulator.internal,
+            org.wso2.eventsimulator.*
+        </export.package>
+        <import.package>
+            org.osgi.framework.*;version="[1.8.0, 2.0.0)",
+            org.wso2.carbon.kernel;version="[5.0.0, 6.0.0)",
```

Comment:
Use properties instead of hardcoding the version. Define version as properties in parent root pom.